usage/camp/src/test/java/org/apache/brooklyn/camp/brooklyn/DslAndRebindYamlTest.java

```diff
@@ -124,6 +126,19 @@ public void testDslAttributeWhenReady() throws Exception {
     }
 
     @Test
+    public void testDslAttributeWhenReadyPersisted() throws Exception {
```

Comment:
as discussed we want to test where `attributeWhenReady` is used in an entity spec, e.g. something like

```
name: Test Cluster
services:
- type: org.apache.brooklyn.entity.stock.DynamicCluster
  id: test-cluster
  initialSize: 1
  memberSpec:
    $brooklyn:entitySpec:
      type: org.apache.brooklyn.entity.test.TestEntity
      brooklyn.config:
        test.confName: $brooklyn:component("test-cluster").attributeWhenReady("sensor")
```

with this you could then in your test set `test-cluster` sensor `sensor` e.g. to `foo` and then ensure that `foo` is *not* being written (ie not in the `testConfNamePersistedState`)